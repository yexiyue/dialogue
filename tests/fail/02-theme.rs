use dialogue_core::Asker;
#[derive(Asker)]
#[asker(theme = "abc")]
struct User{
    name: String,
    age: u8,
}

fn main() {}
