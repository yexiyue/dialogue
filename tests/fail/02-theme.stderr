error[E0433]: failed to resolve: use of undeclared crate or module `abc`
 --> tests/fail/02-theme.rs:2:10
  |
2 | #[derive(Asker)]
  |          ^^^^^ use of undeclared crate or module `abc`
  |
  = note: this error originates in the derive macro `Asker` (in Nightly builds, run with -Z macro-backtrace for more info)
